lib/rules/prettier.js

```diff
@@ -26,10 +26,10 @@ export default class Prettier extends Rule {
 
     if (!prettier) {
       // Prettier is expensive to load, so only load it if needed.
-      prettier = require("prettier");
+      prettier = require("@prettier/sync");
```

Comment:
Using `@prettier/sync`, as I don't think visitors can have `async` methods at the moment?

Reply:
As far as I can tell, they can't.